### PR TITLE
Add additional configurations

### PR DESCRIPTION
**Pull request by shannonssd**
- Add `authlib` library
- Add `mypy` django config
- Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
     * ```pyproject.toml```:
         * Line 2: ```name = "django-boilerplate"```
         * Line 4: ```description = "PeerPower Django Template"```
+        * Line 41: ```django_settings_module = "app_template.project.settings.base"```
```
Comment (jozeffrtus) [inline README.md:25]: Maybe `settings.dev` instead. `base` is not supposed to be directly used, it is only place for defining settings common for dev, test and live environments.

File: pyproject.toml [hunk 2]
```diff
@@ -32,3 +33,9 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.black]
 line-length = 100
+
+[tool.mypy]
+plugins = ["mypy_django_plugin.main"]
+
+[tool.django-stubs]
+django_settings_module = "app_template.project.settings.base"
```
Comment (jozeffrtus) [inline pyproject.toml:41]: Maybe `settings.dev` instead. `base` is not supposed to be directly used, it is only place for defining settings common for dev, test and live environments.

**Review by jozeffrtus — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (shannonssd) [reply pyproject.toml:41]: Noted, will change it. I think the `investor-reporting` repo needs to be changed as well.
https://github.com/peerpower/investor-reporting/blob/main/pyproject.toml

Comment (chawaponake) [reply pyproject.toml]: @freyadam Please note.

**Review by chawaponake — APPROVED** [review]
LGTM

**Review by jozeffrtus — APPROVED** [review]

**Pull request merged**